src/message-handler/src/MessageHandlerContainer.cpp

```diff
@@ -0,0 +1,23 @@
+#include "MessageHandlerContainer.h"
+#include <cpp-common/CircularQueueStack.h>
+
+ThreadSafeQueue<MessageDTO>& MessageHandlerContainer::getBuzzMsgQueue() {
+    static CircularQueueStack<MessageDTO, 8> s_buzzMsgQueue;
```

Comment:
should we maybe move the queue sizes to a macro or a const?

Reply:
Or even maybe as a compile setting?

Reply:
I'll put a const expression for now